ibm/service/power/data_source_ibm_pi_dhcp.go

```diff
@@ -6,113 +6,103 @@ package power
 import (
 	"context"
 	"fmt"
-
 	"log"
 
-	st "github.com/IBM-Cloud/power-go-client/clients/instance"
+	"github.com/IBM-Cloud/power-go-client/clients/instance"
 	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
-
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
 func DataSourceIBMPIDhcp() *schema.Resource {
 	return &schema.Resource{
 		ReadContext: dataSourceIBMPIDhcpRead,
 		Schema: map[string]*schema.Schema{
-
-			// Required Arguments
+			// Arguments
 			Arg_CloudInstanceID: {
-				Type:         schema.TypeString,
+				Description:  "The GUID of the service instance associated with an account.",
 				Required:     true,
+				Type:         schema.TypeString,
 				ValidateFunc: validation.NoZeroValues,
 			},
 			Arg_DhcpID: {
-				Type:        schema.TypeString,
+				Description: "ID of the DHCP Server.",
 				Required:    true,
-				Description: "The ID of the DHCP Server",
+				Type:        schema.TypeString,
 			},
 
 			// Attributes
 			Attr_DhcpID: {
-				Type:        schema.TypeString,
 				Computed:    true,
-				Description: "The ID of the DHCP Server",
+				Deprecated:  "The field is deprecated, use id instead.",
```

Comment:
This was supposed to be deprecated because `pi_dhcp_id` also has the same value. We cannot use `id` here since it is a combination of `workspace_id/dhcp_id`.

Reply:
@ismirlia @michaelkad  we need to ensure all single data sources follow one standard. I see `id` is set to `workspace_id/resource_id` or just `resource_id`.
I am afraid this will be breaking changes but I will leave it up to you guys how you want to handle them.